tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -137,7 +134,7 @@ describe('Performing an upgrade', () => {
   });
 
   it('should display upgrade page even without upgrade logs', async () => {
-    await loginPage.cookieLogin({
+    await oldNavigationPage.cookieLogin({
```

Comment:
```suggestion
    await loginPage.cookieLogin({
```

Since at this point we are already in the new version, you can use the `cookieLogin` from `loginPage`, there is no need to create a new method in `oldNavigationPage`